pkg/distro/generic/images.go

```diff
@@ -420,6 +428,18 @@ func diskImage(workload workload.Workload,
 
 	img.Filename = t.Filename()
 
+	img.VPCForceSize = t.ImageTypeYAML.DiskImageVPCForceSize
+
+	if img.OSCustomizations.NoBLS {
```

Comment:
Just FTR: this comes from https://github.com/osbuild/images/blob/v0.155.0/pkg/distro/rhel/images.go#L427